assets/controller.yaml

```diff
@@ -47,6 +47,10 @@ spec:
           volumeMounts:
             - name: socket-dir
               mountPath: /var/lib/csi/sockets/pluginproxy/
+          resources:
+            requests:
+              memory: 50Mi
+              cpu: 10m
```

Comment:
I checked the resource usage while running the CSI test suite.

Controller pod:
* Biggest memory consumers: CSI driver and the provisioner, around 12 Mi.
* Biggest CPU consumer: CSI driver, around 5m

Node pod (there was only 1 node scheduleable):
* Biggest memory consumer: CSI driver, around 9 Mi.
* Biggest CPU consumer: CSI driver, around 6m

I'd propose start with **requesting** 25Mi for memory and 10m for CPU. Since we're using `request` and not `limit`, the containers can exceed those numbers if the node has memory available.

Other than that, I don't think the numbers proposed in this PR (50Mi and 10m) are too bad, so I'm not against going with that.

```
$ oc -n openshift-cluster-csi-drivers adm top pod aws-ebs-csi-driver-controller-6b679b85fd-6hvxl  --containers
POD                                              NAME              CPU(cores)   MEMORY(bytes)   
aws-ebs-csi-driver-controller-6b679b85fd-6hvxl   csi-snapshotter   0m           8Mi             
aws-ebs-csi-driver-controller-6b679b85fd-6hvxl   csi-driver        5m           12Mi            
aws-ebs-csi-driver-controller-6b679b85fd-6hvxl   csi-attacher      2m           10Mi            
aws-ebs-csi-driver-controller-6b679b85fd-6hvxl   csi-provisioner   2m           12Mi            
aws-ebs-csi-driver-controller-6b679b85fd-6hvxl   csi-resizer       1m           8Mi             

$ oc -n openshift-cluster-csi-drivers adm top pod aws-ebs-csi-driver-node-t2pg7 --containers
POD                             NAME                        CPU(cores)   MEMORY(bytes)   
aws-ebs-csi-driver-node-t2pg7   csi-liveness-probe          0m           7Mi             
aws-ebs-csi-driver-node-t2pg7   csi-node-driver-registrar   0m           5Mi             
aws-ebs-csi-driver-node-t2pg7   csi-driver                  6m           9Mi   
```

PS: Some containers had no usage of CPU or memory because tests that use that containers were not running (csi-snapshotter) or because the container does little work (node registrar and liveness probe).